<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pedidos</class>
 <widget class="QMainWindow" name="Pedidos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pedidos</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(222, 221, 218);
	border:none;
}
QPushButton {
	background-color:rgb(255, 255, 255);
	border:1px solid rgb(100,100,100);
	color:rgb(50,50,50);
    padding: 8px 16px;
    border-radius: 6px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #D6D2D1;
}

QPushButton:pressed {
    background-color: #52c7ff;
}

QLabel{
		font-weight:500;
}

*{
		font-family:&quot;Inter&quot;, sans-serif;
		
}

QFrame{
		background-color: rgb(222, 221, 218);
		border-radius:10%;
}

QLineEdit {
    background-color: white;
    border: 1px solid gray;
    padding: 4px;
    border-radius: 6px;
    font-size: 12px;
}

QLineEdit:focus {
    border-color: #3daee9;
    outline: none;
}

QLineEdit::hover {
    background-color: #f0f0f0;
}

QTextEdit {
    background-color: white;
    border: 1px solid gray;
    padding: 4px;
    border-radius: 6px;
    font-size: 12px;
}

QTextEdit:focus {
    border-color: #3daee9;
    outline: none;
}

QTextEdit::hover {
    background-color: #f0f0f0;
}

QLabel{
		font-weight:500;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTableWidget" name="tb_Pedidos">
           <property name="styleSheet">
            <string notr="true">QTableView QHeaderView::section {
    font-size: 12px;
    font-weight: bold;
    background-color: lightgray;
    padding: 4px;
    border: none;
    border-bottom: 1px solid gray;
}

QTableView {
    background-color: white;
    border: 1px solid gray;
}

QTableView QScrollBar:vertical {
    width: 14px;
    background-color: lightgray;
}

QTableView QScrollBar::handle:vertical {
    background-color: gray;
    border-radius: 7px;
}

QTableView QScrollBar::add-line:vertical,
QTableView QScrollBar::sub-line:vertical {
    background-color: transparent;
}

QTableView QScrollBar::add-page:vertical,
QTableView QScrollBar::sub-page:vertical {
    background-color: transparent;
}

QTableView QScrollBar:horizontal {
    height: 14px;
    background-color: lightgray;
}

QTableView QScrollBar::handle:horizontal {
    background-color: gray;
    border-radius: 7px;
}

QTableView QScrollBar::add-line:horizontal,
QTableView QScrollBar::sub-line:horizontal {
    background-color: transparent;
}

QTableView QScrollBar::add-page:horizontal,
QTableView QScrollBar::sub-page:horizontal {
    background-color: transparent;
}
</string>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>151</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Descripcion</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Fecha del Pedido</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ID_Cliente</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="3" column="1">
            <widget class="QDateEdit" name="dt_pedido">
             <property name="styleSheet">
              <string notr="true">QDateEdit {
    background-color: white;
    border: 1px solid gray;
    padding: 4px;
    border-radius: 6px;
    font-size: 12px;
}

QDateEdit:focus {
    border-color: #3daee9;
    outline: none;
}

QDateEdit::hover {
    background-color: #f0f0f0;
}

</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Descripci√≥n:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Fecha del pedido:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cb_cliente">
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    background-color: white;
    border: 1px solid gray;
    padding: 4px;
    border-radius: 6px;
    font-size: 12px;
}

QLineEdit:focus {
    border-color: #3daee9;
    outline: none;
}

QLineEdit::hover {
    background-color: #f0f0f0;
}</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="line_Id">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
    background-color: white;
    border: 1px solid gray;
    padding: 4px;
    border-radius: 6px;
    font-size: 12px;
}

QLineEdit:focus {
    border-color: #3daee9;
    outline: none;
}

QLineEdit::hover {
    background-color: #f0f0f0;
}</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Cliente:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextEdit" name="line_Descripcion">
             <property name="maximumSize">
              <size>
               <width>2000</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QTextEdit {
    background-color: white;
    border: 1px solid gray;
    padding: 4px;
    border-radius: 6px;
    font-size: 12px;
}

QTextEdit:focus {
    border-color: #3daee9;
    outline: none;
}

QTextEdit::hover {
    background-color: #f0f0f0;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="bt_Guardar">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:1px solid rgb(100,100,100);
	color:rgb(50,50,50);
    padding: 8px 16px;
    border-radius: 6px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #D6D2D1;
}

QPushButton:pressed {
    background-color: #52c7ff;
}
</string>
               </property>
               <property name="text">
                <string>Guardar</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="bt_Editar">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:1px solid rgb(100,100,100);
	color:rgb(50,50,50);
    padding: 8px 16px;
    border-radius: 6px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #D6D2D1;
}

QPushButton:pressed {
    background-color: #52c7ff;
}
</string>
               </property>
               <property name="text">
                <string>Editar</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="bt_Eliminar">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:1px solid rgb(100,100,100);
	color:rgb(50,50,50);
    padding: 8px 16px;
    border-radius: 6px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #D6D2D1;
}

QPushButton:pressed {
    background-color: #52c7ff;
}
</string>
               </property>
               <property name="text">
                <string>Eliminar</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="bt_Vaciar">
               <property name="styleSheet">
                <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:1px solid rgb(100,100,100);
	color:rgb(50,50,50);
    padding: 8px 16px;
    border-radius: 6px;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #D6D2D1;
}

QPushButton:pressed {
    background-color: #52c7ff;
}
</string>
               </property>
               <property name="text">
                <string>Vaciar Campos</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font-family:&quot;Inter&quot;, sans-serif;
font-size:30px;
font-weight:500;</string>
      </property>
      <property name="text">
       <string>Pedidos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
